### PR TITLE
Initial contribution

### PR DESCRIPTION
**Pull request by michelu89**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,654 @@
-# sds-sdk-js-schematics
-This repository is dedicated to SDS Working Group for developing an SDK for Aspect Model in Javascript
+# SKD JS :: Angular Schematic 👷
```
Comment (atextor) [inline README.md:1]: SKD? I'd say "ESMF JS :: Angular Schematic"?

File: documentation/js-sdk-guide/modules/ROOT/pages/getting-started-guide.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+=== Prerequisites
+
+Install https://nodejs.org/en/download/[node] LTS (long term support) version.
+
+=== Install SAMM Schematic
+
+Run the following command in the desired Angular project folder to add the dependency
+
+[source]
+npm install https://github.com/eclipse-esmf/esmf-sdk-js-schematics/releases/download/vX.X.X/esmf-semantic-ui-schematics-X.X.X.tgz --save-dev
```
Comment (atextor) [inline documentation/js-sdk-guide/modules/ROOT/pages/getting-started-guide.adoc:24]: Can you use a version macro to use an actual released version, e.g.
`npm install .../download/{esmf-sdk-js-schematics-version}/...`
(And have the value set in site.yml in the root docs project)

File: documentation/js-sdk-guide/modules/tooling-guide/images/aspect.png
Comment (atextor) [inline documentation/js-sdk-guide/modules/tooling-guide/images/aspect.png:1]: Update, still contains `bamm`

File: documentation/js-sdk-guide/modules/tooling-guide/images/entity.png
Comment (atextor) [inline documentation/js-sdk-guide/modules/tooling-guide/images/entity.png:1]: Update, still contains `bamm`

File: documentation/js-sdk-guide/modules/tooling-guide/images/schema.aspect-or-entity-selection.png
Comment (atextor) [inline documentation/js-sdk-guide/modules/tooling-guide/images/schema.aspect-or-entity-selection.png:1]: Update, this still contains the legacy Movement namespace

File: documentation/js-sdk-guide/modules/tooling-guide/images/schema.exclude-properties.png
Comment (atextor) [inline documentation/js-sdk-guide/modules/tooling-guide/images/schema.exclude-properties.png:1]: Update, still contains legacy namespace

File: test/models/test-collection-types.ttl [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:org.eclipse.esmf.test:1.0.0#> .
+
+:TestCollectionTypes a samm:Aspect ;
+   samm:properties ( :productTypesList :productTypesSet :productTypesSortedSet :measurements ) ;
+   samm:operations ( ) ;
+   samm:name "TestCollectionTypes" .
```
Comment (atextor) [inline test/models/test-collection-types.ttl:30]: Remove `samm:name` from all models. This was removed in BAMM 2.0.0.

**Review by atextor — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (michelu89) [reply README.md:1]: Its EMSF SDK JS ..

File: documentation/js-sdk-guide/antora.yml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 name: js-sdk-guide
 title: JavaScript SDK
-version: master
+version: 2.0.0
```
Comment (atextor) [inline documentation/js-sdk-guide/antora.yml:3]: You must take care that `main` has a different version than the tags, i.e., IMO this should stay on `master` on the `main` branch.

**Review by atextor — APPROVED** [review]

**Pull request merged**